docs/en/query_language/functions/encoding_functions.md

```diff
@@ -1,5 +1,8 @@
 # Encoding functions
 
+## char
+Accepts multiple arguments of `Number` types. Returns a string, each char of the results corresponds to the ascii character of the input numbers. It'll cast the first byte from the number, if the byte overflows the range of ascii(which is 127), it returns an unrecognized character(�).
```

Comment:
We can construct UTF-8 string with this function. It's not limited for ASCII. I will test...